### PR TITLE
SpriteImplementation&CodeWork

### LINKED ISSUE #1410: Unitologist ERT (Pillgrims)
So currently we have zealot RIGs in game but the pilgrims still spawn with the old exo suits.

Along with no gear on their ship all they get currently is a combat belt with a unitologist knife along with three vendors in the back of the ship. they also do not have a radio or any gear than that mentioned.

They need a little bit of an update.

### PR DESCRIPTION
**Pull request by Snypehunter007**
**Added:**
- Added two new .dmi files for in-hand suit items (lefthand_suits.dmi and righthand_suits.dmi). This is a long overdue change that was just never done.
- Added a black tie to the Director of Mining's closet.
- Added security HUD glasses to each of the personnel lockers in the Brig.
- Added a support weapon cabinet to the Brig Armory in place of the floodlight, it will spawn either a Seeker Rifle or Heavy Pulse Rifle.
- Added four divet magazines to the Brig Armory.
- Added Cre's new bio suit.
- Added Cre's new EVA RIG.
- Added Cre's new gas mask.
- Added Cre's new breath mask.
- Added Cre's new engineer lockers.
- Added Cre's new fridge closet.
- Added Cre's new EVA/02 locker.
- Added new in-inventory sprites for all uniforms that didn't have one.
- Added PCSI armor, gloves, and helmet to the security personnnel lockers. 
- Added the SCAF Elite, Legionnaire, and Sharpshooter RIGs.

**Changed:**
- Changed guncabinets to give a pulse rifle and bolas launcher, but only two mags of each.
- Changed the pathing on the Kellion Grunt's gloves to eliminate redundant code.
- Changed the names of riot RIG's sprites to ds_riot_rig for clarity.
- Reduced the Ishimura bio suit's stats.
- Moved various supply packs to the appropriate supply pack files.

**Removed:**
- Removed security RIGs from Security personnel lockers and moved them to the Armory.
- Removed some old concept boots and gloves from Pandemonium that were never used.
- Removed an old EVA helmet concept that I accidentally left in.
- Removed the chicken suit in-hand sprites.
- Removed the captain's jacket in-hand sprites.
- Removed the captain's tunic in-hand sprites.
- Removed the centcom armor in-hand sprites.
- Removed the old firesuit in-hand sprites.
- Removed the first responder in-hand sprites.
- Removed the great coat in-hand sprites.
- Removed the imperium monk in-hand sprites.
- Removed the johnny coat in-hand sprites.
- Removed the nazi uniform in-hand sprites.
- Removed the holiday priest in-hand sprites.
- Removed the really old RIG helm and suit in-hand sprites.
- Removed the brown_jacket in-hand sprites (they are the same as det_suit, if someone wants to make in-hands for it, be my guest).
- Removed the galoshes in-hand sprites.
- Removed teargas box from CSECO locker.
- Removed the node room landmark from the Autopsy room.
- Removed the node room landmark from the Medical Supply Storage room.
- Removed the node room landmarks from the EVA Deck.

**Fixed:**
- Fixed the Vintage, Security, Engineering, and Firesuit RIG boots not having proper in-hand animations.
- Fixed the Unitologist ERT having the old voidsuits instead of the new RIG.
- Fixed the Chief Security Officer's jacket having the wrong on-mob sprite.
- Fixed the Kellion lockers having the gear from the Magpie event.
- Fixed the Deliverance lockers being empty.
- Fixed the Bluespace Technician spawning with a Javelin gun.
- Fixed Escape Pod Nine having the wrong area.
- Fixed the firesuit helmet in-inventory sprites.

**Misc:**
- Disabled the old bio suits from being spawned (for humans) until Cre has time to update them.

Fixes #1410, #1431,

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/l3closet.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 /obj/structure/closet/l3closet/general/WillContain()
 	return list(
-		/obj/item/clothing/suit/storage/hooded/ds_bio_suit,
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/l3closet.dm:15]: I just made this a month or two ago, using new sprites from cre77

File: code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm [hunk 2]
```diff
@@ -10,20 +10,12 @@
 	icon_opened = "base"
 
 /obj/structure/closet/secure_closet/guncabinet/WillContain()
-	var/list/stuff = list(/obj/random/gun_security  = 1, /obj/random/ammo/security = 2)
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm:13]: This was an intentional design to add variety to security gameplay. Please revert guncabinet changes in this PR

File: code/datums/outfits/jobs/security.dm [hunk 3]
```diff
@@ -4,8 +4,7 @@
 	l_ear = /obj/item/device/radio/headset/headset_sec
 	glasses = /obj/item/clothing/glasses/sunglasses/sechud
 	belt = /obj/item/weapon/storage/belt/holster/security
-	gloves = /obj/item/clothing/gloves/combat
```
Comment (NanakoAC) [inline code/datums/outfits/jobs/security.dm:7]: why no gloves? Security having gloves is canon in all of their art, including the new armor we're planning

https://static.wikia.nocookie.net/deadspace/images/4/4d/2l916xw.jpg/revision/latest/scale-to-width-down/1000?cb=20130316233827

File: code/datums/supplypacks/mining.dm [hunk 4]
```diff
@@ -31,3 +31,122 @@
 	containername = "\improper Planet Cracker suits"
 	access = access_mining
 
+
+/decl/hierarchy/supply_pack/mining/mining_cutter
```
Comment (NanakoAC) [inline code/datums/supplypacks/mining.dm:35]: I've been asked and answered about these before. All of these guns had their supplypacks contained in their own files for ease of centralised rebalancing. It saves time and effort making tweaks to things

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 5]
```diff
@@ -66,62 +66,100 @@
...
+		/obj/item/weapon/gun/projectile/divet,
+		/obj/item/ammo_magazine/divet = 2,
 		/obj/item/weapon/reagent_containers/spray/pepper,
 		/obj/item/weapon/melee/baton/loaded,
-		/obj/item/weapon/storage/box/holobadge,
+		/obj/item/taperoll/police,
+		/obj/item/device/hailer,
 		/obj/item/device/holowarrant,
-		/obj/random/tool,
-		/obj/item/weapon/rig/riot
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:95]: Have you discussed this with west? The placement of the riot rig there was kind of temporary, but its future role hasn't been entirely determined

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm [hunk 6]
```diff
@@ -30,51 +30,39 @@
 		if ("small")
 			new /obj/item/weapon/tank/emergency/oxygen(src)
 			new /obj/item/weapon/tank/emergency/oxygen(src)
+			new /obj/item/weapon/rig/emergency(src)
 			new /obj/item/clothing/mask/breath(src)
 			new /obj/item/clothing/mask/breath(src)
-			new /obj/item/clothing/suit/space/void(src)
-			new /obj/item/clothing/head/helmet/space/void (src)
-			if (prob(70))
-				new /obj/item/weapon/tool/tape_roll(src)
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/utility_closets.dm:38]: tape rolls being present in emergency lockers is important, as they can be used for breach patching

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm [hunk 7]
```diff
@@ -30,51 +30,39 @@
 		if ("small")
 			new /obj/item/weapon/tank/emergency/oxygen(src)
 			new /obj/item/weapon/tank/emergency/oxygen(src)
+			new /obj/item/weapon/rig/emergency(src)
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/utility_closets.dm:33]: why giving out a rig in every option, that seems far too much

File: code/modules/clothing/spacesuits/rig/suits/civilian.dm [hunk 8]
```diff
@@ -17,3 +17,33 @@
 		/obj/item/rig_module/healthbar,
 		/obj/item/rig_module/storage/light
 		)
+
+/obj/item/weapon/rig/emergency
+	name = "Emergency RIG"
+	desc = "A light, emergency rig for use by non-qualified personnel in the case of emergency decompression."
+	icon_state = "eva_suit"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 50)
```
Comment (NanakoAC) [inline code/modules/clothing/spacesuits/rig/suits/civilian.dm:25]: a spaceproof suit should have some armor. not much, maybe 20 in all fields

File: code/modules/clothing/spacesuits/rig/suits/civilian.dm [hunk 9]
```diff
@@ -17,3 +17,33 @@
...
+	online_slowdown = 1
```
Comment (NanakoAC) [inline code/modules/clothing/spacesuits/rig/suits/civilian.dm:26]: this is a bit too lightweight. being easy to move in is an advantage of some of the fancier rigs like the Advanced rig, it shouldn't be given out easily

File: code/modules/clothing/spacesuits/rig/suits/riot.dm [hunk 10]
```diff
@@ -1,7 +1,7 @@
 /obj/item/weapon/rig/riot
 	name = "riot rig"
 	desc = "A heavy duty and armoured rig suit, designed for riot control and shipboard disciplinary enforcement. This suit is commonly found among security officers on the Sprawl, and in lesser capacity on private ships and stations."
-	icon_state = "ds_police_rig"
```
Comment (NanakoAC) [inline code/modules/clothing/spacesuits/rig/suits/riot.dm:4]: what's going on here? we had the black riot rig sprites specially made, and we're still discussing the uses of new ones

File: code/modules/clothing/spacesuits/rig/suits/security.dm [hunk 11]
```diff
@@ -1,3 +1,47 @@
+//Ishimura Sec RIGs
+
+
+//PCSI RIG
+/obj/item/weapon/rig/security/pcsi
+	name = "PCSI rig"
+	desc = "A lightweight and flexible armoured rig suit used by CEC shipboard security during crackdowns and for use in hazardous environments."
+	icon_state = "ishi_security_rig"
```
Comment (NanakoAC) [inline code/modules/clothing/spacesuits/rig/suits/security.dm:8]: this sprite is made for an advanced, heavier security rig, the PCSI rig should use a more basic one, this has been hashed out a lot recently

File: code/modules/clothing/suits/armor.dm [hunk 12]
```diff
@@ -20,21 +20,21 @@
...
-	icon_state = "ds_securityarmor"
-	//item_state = "armor"
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO
-	cold_protection = UPPER_TORSO|LOWER_TORSO
-	heat_protection = UPPER_TORSO|LOWER_TORSO
-	armor = list(melee = 45, bullet = 40, laser = 50, energy = 0, bomb = 25, bio = 0, rad = 0)
+/obj/item/clothing/suit/armor/vest/pcsi
+	name = "P.C.S.I officer's vest"
+	desc = "The Concordance Extraction Corporation's standard issue security armor, used on its larger vessels and colonial interests, outfitted with protective polymer plating and a lightweight kevlar web beneath those plates. Deployed in high-risk situations where CEC property and personnel are at risk, usually from piracy or looters."
+	icon_state = "pcsi_armor"
```
Comment (NanakoAC) [inline code/modules/clothing/suits/armor.dm:26]: this sprite, with an underlay, is what we'll be using for the PCSI rig. The armor vest will be something else

File: code/modules/projectiles/guns/ds13/contactbeam.dm [hunk 13]
```diff
@@ -312,24 +312,3 @@
 			new /obj/effect/effect/expanding_circle(location, 0.25, 1. SECONDS)
 			sleep(rand_between(2,4))
 		qdel(src)
-
-
-
-/*
-	Acquisition
-*/
-/decl/hierarchy/supply_pack/mining/contact_energy
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/ds13/contactbeam.dm:321]: see previous comment regarding the placement of these decls.

I've been asked about it and defended their current position, opposed to moving. This does not help work efficiency

**Review by NanakoAC — CHANGES_REQUESTED** [review]
Lots of concerns. Chiefly:

1. The biosuits, me and cre both worked hard on the current implementation and I was very happy with it, would prefer to fix problems rather than replacing them. They are used for both humans and necromorphs, and come in two variants which this odd replacement doesn't have.

Farthermore, the new one looks far too much like a normal medical uniform.

2. Guncabinet changes, please revert. That was by design

3. The whole mess with security rig sprites, there has been a lot of discussion about this lately, and it's probably just communication not catching up. Me and west hammered out an implementation we can agree on, this PR doesn't quite align with that

4. various other misc comments

File: code/modules/clothing/spacesuits/rig/suits/riot.dm (hunk 10 above)
Comment (Snypehunter007) [reply code/modules/clothing/spacesuits/rig/suits/riot.dm:4]: You really didn't read any of the summary post that explains this, did you?

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm (hunk 6 above)
Comment (Snypehunter007) [reply code/game/objects/structures/crates_lockers/closets/utility_closets.dm:38]: Apologies on this one, thought it was the department "Do not cross" tape. I'll put this back in.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 5 above)
Comment (Snypehunter007) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:95]: Yes.

File: code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm (hunk 2 above)
Comment (Snypehunter007) [reply code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm:13]: Got approval from West.

File: code/game/objects/structures/crates_lockers/closets/l3closet.dm (hunk 1 above)
Comment (Snypehunter007) [reply code/game/objects/structures/crates_lockers/closets/l3closet.dm:15]: Implementation was of the item was really bad. Cre also doesn't like them and wants to redo them. For now, they are replaced on the human side by the new sprite.

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm (hunk 7 above)
Comment (Snypehunter007) [reply code/game/objects/structures/crates_lockers/closets/utility_closets.dm:33]: It is a shitty, emergency rig that only has the purpose of keeping the wearer from suffocating. It replaces the emergency helmet and suit and has the same stats as it's counterpart, the standard emergency space suit.

File: code/modules/clothing/spacesuits/rig/suits/civilian.dm (hunk 8 above)
Comment (Snypehunter007) [reply code/modules/clothing/spacesuits/rig/suits/civilian.dm:25]: It is a shitty, emergency rig that only has the purpose of keeping the wearer from suffocating. It replaces the emergency helmet and suit and has the same stats as it's counterpart, the standard emergency space suit.

File: code/datums/supplypacks/mining.dm (hunk 4 above)
Comment (Snypehunter007) [reply code/datums/supplypacks/mining.dm:35]: We have also made it clear before that this is not how the system works and that you should stop doing it. Yet you persist.

File: code/modules/projectiles/guns/ds13/contactbeam.dm (hunk 13 above)
Comment (Snypehunter007) [reply code/modules/projectiles/guns/ds13/contactbeam.dm:321]: We have also made it clear before that this is not how the system works and that you should stop doing it. Yet you persist.

File: code/modules/clothing/spacesuits/rig/suits/civilian.dm (hunk 9 above)
Comment (Snypehunter007) [reply code/modules/clothing/spacesuits/rig/suits/civilian.dm:26]: It has no stats practically and has the same online and offline slowdown as the firesuit.

It is a glass cannon.

**Pull request merged**